{
  diagnosticLoggerEntries: [
    {
      Level: error,
      Message: Reentrant log event detected. Logging when inside the scope of another log event can cause a StackOverflowException. LogEventInfo.Message: message from OnEvaluating
    }
  ],
  Envelopes: [
    {
      Header: {
        event_id: Guid_1,
        sdk: {
          name: sentry.dotnet
        }
      },
      Items: [
        {
          Header: {
            type: event
          },
          Payload: {
            Source: {
              Message: {
                Message: message,
                Formatted: message
              },
              Platform: csharp,
              SentryExceptions: [],
              SentryThreads: [],
              Level: error,
              Request: {},
              Contexts: {},
              User: {},
              Environment: production
            }
          }
        }
      ]
    }
  ]
}